sub/dec_sub.c

```diff
@@ -369,6 +369,15 @@ char *sub_get_text(struct dec_sub *sub, double pts, enum sd_text_type type)
     return text;
 }
 
+char *sub_ass_get_extradata(struct dec_sub *sub)
+{
+    if (strcmp(sub->sd->codec->codec, "ass") < 0)
```

Comment:
```suggestion
    if (strcmp(sub->sd->codec->codec, "ass") != 0)
```
Oops, missed that yesterday apparently.